strr-platform-web/app/stores/platformDetails.ts

```diff
@@ -1,15 +1,21 @@
 import { z } from 'zod'
 
 export const useStrrPlatformDetails = defineStore('strr/platformDetails', () => {
+  const { t } = useI18n()
   const {
     addNewEmptyBrand: baseAddNewEmptyBrand,
     getBrandSchema: getPlatformBrandSchema,
     removeBrandAtIndex: baseRemoveBrandAtIndex
   } = useStrrBaseBrand()
 
+  const getPlatformSchema = () => z.object({
```

Comment:
Had to update this because the error message changed and I didnt want to conflict in strata

Reply:
If we keep the i18n path the same then it can stay in the common I think? Platforms can just overwrite the message it wants in the locale file

Reply:
Hmm yeah, my bad I didn't even think of that. We should update the base one to be more generic.